src/main/java/org/folio/circulation/domain/RequestRules.java

```diff
@@ -0,0 +1,65 @@
+package org.folio.circulation.domain;
+
+import java.util.EnumMap;
+
+public class RequestRules {
```

Comment:
Given that this class (was separated from request type and) is intended to represent the decision about which item states allow a request of a certain type to be created, I think we could give this a different name.

My suggestions would be:
* RequestAndItemStatusWhiteList
* ItemStatusWhiteList (in a request specific package)
* RequestTypeItemStatusWhiteList

What do you think?

Reply:
I think RequestTypeItemStatusWhiteList makes most sense.